packages/jupyter-ai-magics/jupyter_ai_magics/providers.py

```diff
@@ -365,6 +370,15 @@ class SmEndpointProvider(BaseProvider, SagemakerEndpoint):
     name = "Sagemaker Endpoint"
     models = ["*"]
     model_id_key = "endpoint_name"
+    # This all needs to be on one line of markdown, for use in a table
+    model_help = (
+        "SageMaker endpoint names are created when you deploy a model. "
+        "For more information, see "
+        '["Create your endpoint and deploy your model"]'
+        "(https://docs.aws.amazon.com/sagemaker/latest/dg/realtime-endpoints-deployment.html) "
+        "in the SageMaker documentation."
+    )
+
```

Comment:
Similarly, I'd prefer:

> The SageMaker Endpoints provider expects a model's endpoint name to be passed via `model_id`, the region name via `region_name`, the response schema via `response_schema`, and the response path via `response_path`. For more information, see the documentation on [SageMaker Endpoints](https://docs.aws.amazon.com/sagemaker/latest/dg/realtime-endpoints-deployment.html).

There should also be an additional section in the user documentation that discusses how to use SMEP via Jupyter AI; there is currently no documentation on exactly what the "request schema" and "response path" are. The help string above should link to these sections, e.g. "the [response path]\(LINK\) via ...".